src/background.js

```diff
@@ -1777,6 +1763,10 @@ function init() {
 					return account.getUser()
 						.then(account.getUserSettings);
 				}
+				// Set the default ghostery blocking settings
+				if (globals.JUST_INSTALLED) {
+					setGhosteryDefaultBlocking();
+				}
```

Comment:
@zarembsky won't this overwrite settings of an already logged-in user? Think of this scenario:
1. User is logged into the website, and already has synced blocking settings
2. User installs a fresh copy of Ghostery
3. This will set default blocking overwriting the user's saved blocking settings.

Reply:
Thanks. I meant else, but lost it somehow.